app/views/directives/resource-service-bindings.html

```diff
@@ -1,12 +1,13 @@
 <div class="mar-bottom-xl" ng-if="$ctrl.showBindings">
-  <h3>Service Bindings</h3>
-  <overview-service-binding
+  <h3>Bindings</h3>
+  <service-binding
     ng-repeat="binding in $ctrl.bindings track by (binding | uid)"
     namespace="$ctrl.projectContext.projectName"
     binding="binding"
+    ref-api-object="$ctrl.apiObject"
     service-classes="$ctrl.serviceClasses"
     service-instances="$ctrl.serviceInstances">
-  </overview-service-binding>
+  </service-binding>
   <div ng-if="($ctrl.bindableServiceInstances | size) && ({resource: 'bindings', group: 'servicecatalog.k8s.io'} | canI : 'create')">
```

Comment:
@spadgett this ```bindings``` looks like it falls into the category of... replace with ```serviceinstancecredentials``` ? looks like there's another ```bindings``` reference around line 23ish too.

If replace is the case for those ```bindings``` do we have a related issue, or is there a PR that's already in progress? Found a couple of more instances in other files...

Reply:
Yes.

We have #2048 to update the names in other places.